### PR TITLE
TSWEB-697 add tests for ConsultingServ

### PR DESCRIPTION
**Pull request by AnnaSlisenko**

### CODE REVIEW

File: ui_tests_playwright/automation/test/ui/services/ConsultingService.test.ts [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+		'Improve code quality and release a quality product;',
+		'Improve motivation and engineering culture;',
+		'Optimize infrastructure costs;',
+		'Remove production issues and keep the environment stable.',
+	];
+
+	expect(allSectionTitles.sort()).toEqual(testData.sort());
+});
+
+test("Check section titles  in 'Development process audit' container from the 'Consulting service' block @Regression @ConsultingService @TSWEB-697", async () => {
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/services/ConsultingService.test.ts:146]: double spaces in test naming: 
...titles  in...

File: ui_tests_playwright/automation/test/ui/services/ConsultingService.test.ts [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+		'Development\nprocess',
+		'Release process /\nIntegration layer',
+		'QA\nprocess',
+		'Business analytics\nprocess',
+	];
+
+	expect(allSectionTitles.sort()).toEqual(testData.sort());
+});
+
+test("Check section titles  in 'Software audit' container from the 'Consulting service' block @Regression @ConsultingService @TSWEB-697", async () => {
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/services/ConsultingService.test.ts:160]: s

File: ui_tests_playwright/automation/test/ui/services/ConsultingService.test.ts [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+	await carouselButtonNext.click();
+
+	await expect(carouselButtonPrev).toHaveAttribute('data-disabled', 'false');
+	await expect(carouselButtonNext).toHaveAttribute('data-disabled', 'false');
+	await carouselButtonPrev.click({delay: 1000});
+
+	await expect(carouselButtonPrev).toHaveAttribute('data-disabled', 'true');
+	await expect(carouselButtonNext).toHaveAttribute('data-disabled', 'false');
+
+	const clickCount = allSectionTitles.length - 1;
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/services/ConsultingService.test.ts:205]: I don't see the reason to create this variable.

**Review by VladFedash — CHANGES_REQUESTED** [review]

**Comment by AnnaSlisenko** [comment]
Obsolete

**Pull request closed without merging**